diff --git a/xilinx/arty_a7_100/tcl/board.tcl b/xilinx/arty_a7_100/tcl/board.tcl
index 552d70f..70635da 100644
--- a/xilinx/arty_a7_100/tcl/board.tcl
+++ b/xilinx/arty_a7_100/tcl/board.tcl
@@ -1,5 +1,5 @@
 # See LICENSE for license details.
 set name {arty-a7-100}
 set part_fpga {xc7a100ticsg324-1L}
-set part_board {digilentinc.com:arty-a7-100:part0:1.1}
+set part_board {digilentinc.com:arty-a7-100:part0:1.0}
 set bootrom_inst {rom}
